public/usage-examples/graphics/draw_triangle_on_bitmap/draw_triangle_on_bitmap-1-mountain-peak-oop.cs

```diff
@@ -0,0 +1,46 @@
+using SplashKitSDK;
+
+namespace DrawTriangleOnBitmap
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a window and bitmap for the mountain scene
+            Window window = new Window("Window", 400, 300);
+            Bitmap bitmap = new Bitmap("mountain", 400, 300);
+            
+            // Fill background with light blue color
+            bitmap.Clear(Color.LightBlue);
```

Comment:
Currently using splashkit functions incorrectly, should be SplashKit.ClearBitmap(bitmap, SplashKit.ColorLightBlue()); to correctly use SplashKit functions